apache-camel-mixin/dashboards/apache-camel-micrometer.json

```diff
@@ -0,0 +1,1909 @@
+        "reduceOptions": {
+          "calcs": [
+            "lastNotNull"
+          ],
+          "fields": "",
+          "values": false
+        },
+        "textMode": "auto"
+      },
+      "pluginVersion": "9.1.0",
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "${datasource}"
+          },
+          "editorMode": "code",
+          "exemplar": false,
+          "expr": "sum(CamelRoutesRunning_routes{namespace=\"$namespace\",camelContext=\"$context\"}) / sum(CamelRoutesAdded_routes{namespace=\"$namespace\",camelContext=\"$context\"})*100",
+          "format": "time_series",
+          "hide": false,
+          "instant": true,
+          "legendFormat": "Running",
+          "range": false,
+          "refId": "RunningRoutesPercentage"
+        },
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "${datasource}"
+          },
+          "editorMode": "code",
+          "exemplar": false,
+          "expr": "max(CamelRoutesAdded_routes{namespace=\"$namespace\",camelContext=\"$context\"})",
+          "format": "time_series",
+          "hide": false,
+          "instant": true,
+          "legendFormat": "Routes",
+          "range": false,
+          "refId": "RoutesCount"
+        }
+      ],
+      "title": "Routes Status",
+      "transformations": [],
+      "type": "stat"
+    },
+    {
+      "datasource": {
+        "type": "prometheus",
+        "uid": "${datasource}"
+      },
+      "description": "Shows completed exchanges of the camel context in the time range",
+      "fieldConfig": {
+        "defaults": {
+          "color": {
+            "fixedColor": "rgb(31, 120, 193)",
+            "mode": "thresholds"
+          },
+          "decimals": 0,
+          "mappings": [],
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "blue",
+                "value": null
+              }
+            ]
+          },
+          "unit": "short"
+        },
+        "overrides": []
+      },
+      "gridPos": {
+        "h": 4,
+        "w": 4,
+        "x": 7,
+        "y": 1
+      },
+      "id": 212,
+      "links": [],
+      "maxDataPoints": 100,
+      "options": {
+        "colorMode": "none",
+        "graphMode": "area",
+        "justifyMode": "center",
+        "orientation": "horizontal",
+        "reduceOptions": {
+          "calcs": [
+            "lastNotNull"
+          ],
+          "fields": "",
+          "values": false
+        },
+        "textMode": "auto"
+      },
+      "pluginVersion": "9.1.0",
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "${datasource}"
+          },
+          "editorMode": "code",
+          "expr": "sum(increase(CamelExchangesTotal_total{namespace=\"$namespace\",camelContext=\"$context\"}[$__range]))",
```

Comment:
There are a few places where you use `rate`, `irate`, or `increase` with `$__range`.

It looks like in each of these cases it is intended to be for "the selected time range" rather than a timeseries showing a rolling interval, for which you should use `$__rate_interval`. See [target-rate-interval-rule](https://github.com/grafana/dashboard-linter/blob/main/docs/rules/target-rate-interval-rule.md).

Not a problem, just an observation, and it would require a dashboard linter [exclusion](https://github.com/grafana/dashboard-linter/blob/main/docs/index.md#exclusions-and-warnings).